python/GafferSceneUITest/OptionInspectorTest.py

```diff
@@ -812,5 +812,56 @@ def assertRenderPassEditResults( renderPass, option ) :
 				with self.subTest( renderPass = renderPass, option = option ) :
 					assertRenderPassEditResults( renderPass, option )
 
+	def testDefaultValueMetadata( self ) :
+
+		customOptions = GafferScene.CustomOptions()
+
+		editScope = Gaffer.EditScope()
+		editScope.setup( customOptions["out"] )
+		editScope["in"].setInput( customOptions["out"] )
+
+		Gaffer.Metadata.deregisterValue( "option:test:enabled", "defaultValue" )
```

Comment:
What's this for? Would it be better in `tearDown` or `addCleanup`? At the moment we'll leak the registration if the test fails...

Reply:
Ah! `addCleanup()` is a handy one. I've now used that here: aa58904